test/expected/10-utilities.out

```diff
@@ -10,11 +10,17 @@ SELECT master_create_distributed_table('sharded_table', 'id');
 
 -- COPY is not supported with distributed tables
```

Comment:
Maybe just remove these tests since COPY is actually supported now, although it might be useful to add a test to copy-utility.source where we try to COPY into a distributed table without shards.